### ISSUE TITLE
Waterline 0.11 upgrade

### ISSUE THREAD

**Issue by shamasis** [postmanlabs/sails-mysql-transactions#33]

**Comment by guretno** [comment]
The waterline and sails-mysql upgrade to 0.11 might able to solve this slow performance bug. https://github.com/balderdashy/sails-mysql/issues/161

**Comment by shamasis** [comment]
Yes. I've attempted to merge the new upstream. There were quite a few merge conflicts. Hence waiting for an opportune time to do it.
